testutil/promrated/prometheus.go

```diff
@@ -41,6 +81,73 @@ func serveMonitoring(addr string) error {
 	return errors.Wrap(server.ListenAndServe(), "failed to serve prometheus metrics")
 }
 
+func getPubkeyToClusterInfo(ctx context.Context, promAuth string) (map[string]prometheus.Labels, error) {
```

Comment:
`getPubkeyToClusterInfo` name can improve, what about `getClusters` or `getClustersFromProm`

Reply:
also add godoc please

Reply:
rather return a proper type instead of map of strings. Also remember that clusters contain multiple validators. Suggest:
```
type Validator struct {
  PubkeyHex string
  ClusterName string
  ClusterHash string
  ClusterPeer string
}
```

Reply:
> `getPubkeyToClusterInfo` name can improve, what about `getClusters` or `getClustersFromProm`

Yeah this naming is what I was used to for map types, can change it